src/qubx/core/interfaces.py

```diff
@@ -403,7 +417,7 @@ def cancel_order(self, order_id: str) -> None:
 class IUniverseManager:
     """Manages universe updates."""
 
-    def set_universe(self, instruments: list[Instrument]):
+    def set_universe(self, instruments: list[Instrument], skip_callback: bool = False):
         """Set the trading universe.
```

Comment:
Q: do we really need skip_callback parameter ?

Reply:
I agree, it's something that I don't like as well. Added this temporarily, will think how to improve. I was thinking that I don't want the initial instruments to go through the on_universe and I want them to be available in the on_start method. And this parameter was related to this choice. But there is likely some more elegant solution.